deepchem/feat/dft_featurizer.py

```diff
@@ -0,0 +1,289 @@
+from __future__ import annotations
```

Comment:
this file has been moved to deepchem/data

Reply:
Make sure to delete this file from the PR before final review